hooks/post-checkout

```diff
@@ -3,7 +3,14 @@
 set -euo pipefail
 
 EJSON2ENV_PLUGIN_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")"; cd ..; pwd)
-EJSON2ENV_BIN_PATH="$EJSON2ENV_PLUGIN_ROOT/ejson2env"
+arch="${BUILDKITE_PLUGIN_EJSON2ENV_ARCH:-}"
```

Comment:
I'm not sure I understand the intent of this arch parameter. If a value is provided is seems to be overridden with in the else block below.

Reply:
Yeah, that was unusable. Removed.